config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml

```diff
@@ -1086,12 +1086,12 @@ spec:
                     type: boolean
                 type: object
               grpcPort:
-                description: '(Optional) The database port (`--port` CLI parameter
+                description: '(Optional) The database port (`--listen-addr` CLI parameter
```

Comment:
Are these changes backwards compatible?

Reply:
Yes. I have only changed the description and not the field so it is backward compatible.